ffmpeg_plugin/README.txt

```diff
@@ -1,24 +1,40 @@
 # svt-hevc ffmpeg plugin installation
 
+For Linux:
 1. Build and install SVT-HEVC 
 - git clone https://github.com/OpenVisualCloud/SVT-HEVC
 - cd SVT-HEVC/Build/linux
 - ./build.sh debug|release
 - cd debug|release
 - make install
+- export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/lib
+- export PKG_CONFIG_PATH=${PKG_CONFIG_PATH}:/usr/local/lib/pkgconfig
+
+For Windows:
+0. Setup environment
+- Install Visual Studio 2017 or 2019
+- Install msys2-x86_64 to C:\msys64 following http://www.msys2.org/
+- Run "C:\msys64\mingw64.exe" to open a console
+- Install the required SW by "pacman -S make perl diffutils cmake yasm gcc pkgconfig git" in the console
```

Comment:
```suggestion
- Install the required SW by "pacman -S --needed make perl diffutils cmake yasm gcc pkgconfig git" in the console
```

Are all of these packages needed? I'm not sure if perl is an explicit needed plus gcc would not be needed if you are using VS2017 or 2019